### PR TITLE
[Organizations] Link empty Manage Organizations page to the Transform flow

### PR DESCRIPTION
**Pull request by scottbommarito**
https://github.com/NuGet/NuGetGallery/issues/5691

Also some small UI fixes to the transform page

Manage organizations page:

image

Transform page:

image

### CODE REVIEW

**Comment by scottbommarito** [comment]
@anangaur let me know what you think would look better on the page

**Comment by anangaur** [comment]
First of all. Not related to this but the breadcrumb looks weird. Too big. The arrow needs to change from the > to another icon.

As for the transform option, we need 3 things:

1. The transform option should not be a button but a linked button:
image

2. There needs to be a confirm dialog as this is an irreversible process:
image

3. There needs to a small fix on the transform page:
image

**Comment by scottbommarito** [comment]
@anangaur what is the change on 3? I don't see anything different

**Comment by scottbommarito** [comment]
@anangaur regarding 2, I'm also concerned that the text you have provided will scare users into thinking that the "transform this account into an organization" button actually performs the transformation. I think the text you're proposing would be better off on the transform form page itself.

**Comment by anangaur** [comment]
Looks like I added wrong picture for 3. Will add the right one soon.

For 2, I am fine with your suggestion. Can you make this a warning banner on the transform page at the top itself?

**Comment by scottbommarito** [comment]
@anangaur I have updated this PR and the screenshots. Any thoughts?

**Comment by scottbommarito** [comment]
@anangaur see updated screenshot

**Comment by anangaur** [comment]
Some fixes required:
1. Left align the breadcrumb
2. Remove the account name
3. Increase the spacing/padding between the warning and the rest of the contents.

The warning seems too big but i do not have any better suggestion as of now. Lets see if people have better inputs in tomorrow' bugbash.

image

**Comment by scottbommarito** [comment]
@anangaur see updated screenshot

File: src/NuGetGallery/Views/Users/Organizations.cshtml [hunk 1]
```diff
@@ -102,6 +102,9 @@
                         <input type="submit" class="btn btn-primary" value="Add new organization" />
                     </form>
                 </div>
+                <div class="subtitle text-center">
+                    <a href="@Url.TransformAccount()">Transform your account into an organization</a>
+                </div>
```
Comment (chenriksson) [inline src/NuGetGallery/Views/Users/Organizations.cshtml]: With the new Org flow, we already updated top right menu to always show the Organizations view link?

Comment (scottbommarito) [reply src/NuGetGallery/Views/Users/Organizations.cshtml]: Yup!

File: src/NuGetGallery/Views/Users/TransformToOrganization.cshtml [hunk 2]
```diff
@@ -15,9 +15,27 @@
...
+                    </span>
+                    <ul class="transform-alert-list">
+                        <li>
+                            You will no longer be able sign in to @CurrentUser.Username.
+                        </li>
+                        <li>
+                            All permissions to manage @CurrentUser.Username and the packages it owns will be transferred to the administrator you choose here.
+                        </li>
+                        <li>
+                            Existing scoped API keys for this account will be transferred to the administrator. Full access API keys (legacy API keys without expiry) will not be transferred and will be deleted.
```
Comment (shishirx34) [inline src/NuGetGallery/Views/Users/TransformToOrganization.cshtml]: may be put ... will not be transferred and will be deleted in bold?

Comment (scottbommarito) [reply src/NuGetGallery/Views/Users/TransformToOrganization.cshtml]: I like that suggestion! Will do.

**Comment by scottbommarito** [comment]
per @anangaur feedback to apply before merging: increase margin between breadcrumb and image in empty page

**Pull request merged**